src/components/video.js

```diff
@@ -132,14 +164,51 @@ class Video extends React.Component {
           id='remoteVideo'
           ref={video => (this.remoteVideo = video)}
         />
+
+        <div className='controls'>
         <button
-          className='share-screen-btn'
+          className='control-btn'
           onClick={() => {
             this.getDisplay();
           }}
         >
           <ShareScreenIcon />
         </button>
+
+
+        <button
+        className='control-btn'
+          onClick={() => {
+            this.setAudioLocal();
+          }}
+        >
+          {
+            this.state.micState?(
+              <MicOnIcon></MicOnIcon>
```

Comment:
This is really small but you can just use ```<MicOnIcon/>```

Reply:
ok